neqo-transport/src/path.rs

```diff
@@ -524,12 +524,18 @@ impl Path {
     ) -> Self {
         let iface_mtu = match mtu::interface_and_mtu(remote.ip()) {
             Ok((name, mtu)) => {
-                qdebug!("Outbound interface {name} has MTU {mtu}");
+                qdebug!(
+                    "Outbound interface {name} for destination {ip} has MTU {mtu}",
+                    ip = remote.ip()
+                );
                 stats.pmtud_iface_mtu = mtu;
                 Some(mtu)
             }
             Err(e) => {
-                qwarn!("Failed to determine outbound interface: {e}");
+                qwarn!(
+                    "Failed to determine outbound interface for destination {}: {e}",
+                    remote.ip()
```

Comment:
```suggestion
                    "Failed to determine outbound interface for destination {ip}: {e}",
                    ip = remote.ip()
```

For the sake of consistency with the `qwarn` above. Or remove the reassignment above.